### PR TITLE
Configure Database Integration

### PR DESCRIPTION
**Pull request by yonas-seyoum**

### CODE REVIEW

**Review by leulsege — APPROVED** [review]

**Review by leulsege — COMMENTED** [review]
first pass all checks

File: services/service-core/package.json [hunk 1]
```diff
@@ -22,10 +22,14 @@
...
   "dependencies": {
     "@nestjs/common": "^10.0.0",
+    "@nestjs/config": "^3.0.0",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "@nestjs/typeorm": "^10.0.0",
+    "pg": "^8.11.2",
     "reflect-metadata": "^0.1.13",
-    "rxjs": "^7.8.1"
+    "rxjs": "^7.8.1",
```
Comment (peterrogov) [inline services/service-core/package.json:31]: What is this for?

Comment (peterrogov) [inline services/service-core/package.json:31]: What is this for?

File: services/service-core/src/app.module.ts [hunk 2]
```diff
@@ -13,10 +13,29 @@
...
+  imports: [
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule.forRoot({
+        isGlobal: true,
+        envFilePath: ".local.env"
+      })],
+      useFactory: (configService: ConfigService) => ({
+        type: 'postgres',
+        host: configService.get('DB_HOST'),
+        port: +configService.get<number>('DB_PORT'),
```
Comment (peterrogov) [inline services/service-core/src/app.module.ts:31]: Better to use Number(configService.get<number>('DB_PORT')). Use of `+` is technically correct but very obscure and unclear

**Review by peterrogov — CHANGES_REQUESTED** [review]

**Review by leulsege — COMMENTED** [review]
So, when will you explain to us all of what you do yoni

**Review by leulsege — APPROVED** [review]

**Review by peterrogov — APPROVED** [review]

**Review by peterrogov — APPROVED** [review]

**Pull request closed without merging**